pkg/pod/v1alpha1/pod.go

```diff
@@ -0,0 +1,111 @@
+package v1alpha1
```

Comment:
Please use the standard folder structure
`pkg/kubernetes/pod/v1alpha1`

Reply:
done